main/settings.py

```diff
@@ -155,3 +156,8 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = "/static/"
+
+STATIC_ROOT = os.path.join(BASE_DIR, ‘static’)
+
+# Activate Django-Heroku.
+django_heroku.settings(locals())
```

Comment:
You can try something like this 
```suggestion
from socket import gethostname

HOSTNAME = gethostname()
if HOSTNAME == 'open-source-programs-backend.herokuapp.com':
    import django_heroku
    django_heroku.settings(locals())
```

Reply:
@devkapilbansal Why use this when django_heroku doesn't affect the development environment?

Reply:
What's the point of setting config if we are using it in development. However, we can ignore this :confused: 
What's not broken should not be fixed